webapi/src/main/scala/org/knora/webapi/responders/admin/ProjectsResponderADM.scala

```diff
@@ -176,14 +172,16 @@ trait ProjectsResponderADM {
   ): Task[ProjectOperationResponseADM]
 
   def projectDataGetRequestADM(id: ProjectIdentifierADM, user: UserADM): Task[ProjectDataGetResponseADM]
+
 }
 
 final case class ProjectsResponderADMLive(
-  triplestoreService: TriplestoreService,
-  messageRelay: MessageRelay,
-  iriService: IriService,
-  cacheServiceSettings: CacheServiceSettings,
-  implicit val stringFormatter: StringFormatter
+  private val triplestoreService: TriplestoreService,
```

Comment:
What is the reason behind adding `private val` to all these parameters? Is it to find unused occurrences later?